cmd/postgres_exporter/probe.go

```diff
@@ -62,27 +61,27 @@ func handleProbe(logger log.Logger) http.HandlerFunc {
 
 		// TODO(@sysadmind): Timeout
 
-		probeSuccessGauge := prometheus.NewGauge(prometheus.GaugeOpts{
-			Name: "probe_success",
-			Help: "Displays whether or not the probe was a success",
-		})
-		probeDurationGauge := prometheus.NewGauge(prometheus.GaugeOpts{
-			Name: "probe_duration_seconds",
-			Help: "Returns how long the probe took to complete in seconds",
-		})
+		// probeSuccessGauge := prometheus.NewGauge(prometheus.GaugeOpts{
+		// 	Name: "probe_success",
+		// 	Help: "Displays whether or not the probe was a success",
+		// })
+		// probeDurationGauge := prometheus.NewGauge(prometheus.GaugeOpts{
+		// 	Name: "probe_duration_seconds",
+		// 	Help: "Returns how long the probe took to complete in seconds",
+		// })
```

Comment:
Let's just delete this. We don't need to expose this stuff.

We already have a `postgres_up` metric.